src/components/dialogs/two-windings-transformer-creation-dialog.js

```diff
@@ -0,0 +1,607 @@
+            '2wt-series-reactance',
+            validateField(seriesReactance, {
+                isFieldRequired: true,
+                isFieldNumeric: true,
+            })
+        );
+
+        tmpErrors.set(
+            '2wt-magnetizing-conductance',
+            validateField(magnetizingConductance, {
+                isFieldRequired: true,
+                isFieldNumeric: true,
+            })
+        );
+
+        tmpErrors.set(
+            '2wt-magnetizing-susceptance',
+            validateField(magnetizingSusceptance, {
+                isFieldRequired: true,
+                isFieldNumeric: true,
+            })
+        );
+
+        tmpErrors.set(
+            '2wt-rated-voltage-side-1',
+            validateField(ratedVoltage1, {
+                isFieldRequired: true,
+                isFieldNumeric: true,
+            })
+        );
+
+        tmpErrors.set(
+            '2wt-rated-voltage-side-2',
+            validateField(ratedVoltage2, {
+                isFieldRequired: true,
+                isFieldNumeric: true,
+            })
+        );
+
+        tmpErrors.set(
+            'voltage-level1',
+            validateField(voltageLevel1, {
+                isFieldRequired: true,
+            })
+        );
+
+        tmpErrors.set(
+            'bus-bar1',
+            validateField(busOrBusbarSection1, {
+                isFieldRequired: true,
+            })
+        );
+
+        tmpErrors.set(
+            'voltage-level2',
+            validateField(voltageLevel2, {
+                isFieldRequired: true,
+            })
+        );
+
+        tmpErrors.set(
+            'bus-bar2',
+            validateField(busOrBusbarSection2, {
+                isFieldRequired: true,
+            })
+        );
+
+        setErrors(tmpErrors);
+
+        // Check if error list contains an error
+        let isValid =
+            Array.from(tmpErrors.values()).findIndex((err) => err.error) === -1;
+
+        if (isValid) {
+            createTwoWindingsTransformer(
+                studyUuid,
+                twoWindingsTransformerId,
+                twoWindingsTransformerName,
+                seriesResistance,
+                seriesReactance,
+                magnetizingConductance,
+                magnetizingSusceptance,
+                ratedVoltage1,
+                ratedVoltage2,
+                voltageLevel1.id,
+                busOrBusbarSection1.id,
+                voltageLevel2.id,
+                busOrBusbarSection2.id
+            )
+                .then(() => {
+                    handleCloseAndClear();
+                })
+                .catch((errorMessage) => {
+                    displayErrorMessageWithSnackbar({
+                        errorMessage: errorMessage,
+                        enqueueSnackbar: enqueueSnackbar,
+                        headerMessage: {
+                            headerMessageId:
+                                'TwoWindingsTransformerCreationError',
```

Comment:
Missing 'TwoWindingsTransformerCreationError' in fr.json and en.json translations files